source/user-manual/wazuh-dashboard/config-file.rst

```diff
@@ -724,6 +724,105 @@ Set the footer of the PDF reports. To use an empty footer, type a space " " in t
 | **Allowed values** | Any string                 |
 +--------------------+----------------------------+
 
+ISM rollover policy
+-------------------
+
+.. warning::
+
+   Modifying these options has no effect if you do it once the Wazuh dashboard started for the first time. Read about different methods to customize the rollover policy in :doc:`/user-manual/wazuh-indexer/index-life-management`.
+
+ism.rollover.enabled
+^^^^^^^^^^^^^^^^^^^^
+
+Toggles the verification of the rollover policy. If set to ``true``, the rollover policy will be applied to the indices that match the index patterns defined in ``ism.rollover.index_patterns``, only if there isn't any other policy managing these indices.
```

Comment:
What does "verification" mean?

Reply:
When the app starts, it will look for the policy. If it's not present, the app will upload it.

I used "verification" because I couldn't find any better term.

I think this process is explained in the life cycle management guide, but maybe we could add it here too.